testsuite/integration/src/test/xslt/setupIiopSslSupport.xsl

```diff
@@ -44,21 +64,4 @@
             </xsl:otherwise>
         </xsl:choose>
     </xsl:template>
-
-    <!-- Enable SSL support in Jacorb configuration -->
-    <xsl:template match="//*[local-name()='subsystem' and starts-with(namespace-uri(), $jacorbNS)]">
```

Comment:
Not turning it off, I'm setting it up differently in the XSL (a new template a little above this one) to take into consideration the new validations that tomek introduced. Without this change the servers fail to boot due to missing security domain when support-ssl is set to true.